### DISCUSSION TITLE
Hosting on Github pages

### DISCUSSION THREAD

**Discussion by DamoM73** [pradyunsg/furo#610] [Questions] [answered]
Hi all

I am a secondary school computer teacher. I have been working on a Python into course for my student suing MD and Sphinx. I have generated the HTML and used this wonderful theme, but now I'm trying got host it on GitHub pages, and all the theme functionality is gone.

I'm very new to the GitHub space and I would really appreciate any suggestion on who to make this work.

**Comment by DamoM73** [comment, marked as answer]
I worked out what I was doing wrong. I found an excellent video detailing the GitHub action I needed to push the website.

https://www.youtube.com/watch?v=SxkM1SK9YlI

**Reply by pradyunsg** [reply]
Glad you found that! :)
